Sources/Plasma/CoreLib/hsMatrix44.h

```diff
@@ -163,6 +171,9 @@ struct hsMatrix44 {
 
     static hsMatrix44 mult_fpu(const hsMatrix44& a, const hsMatrix44& b);
     static hsMatrix44 mult_sse3(const hsMatrix44& a, const hsMatrix44& b);
+#ifdef HS_BUILD_FOR_APPLE
+    static hsMatrix44 mult_acclerate(const hsMatrix44 &a, const hsMatrix44 &b);\
```

Comment:
trailing slash here that seems to maybe be breaking the Linux build

Reply:
Yep, just saw that and amended the commit

Reply:
Also I misspelled Accelerate